_projects/2023/gnuastro/python_wrappers.md

```diff
@@ -0,0 +1,70 @@
+---
+name: Gnuastro library in Python
+desc: Python wrappers over the Gnuastro C library.
+requirements:
+# Student requirements:
+ - Low-level Python and Numpy (their C library).
+difficulty: low
+issues:
+# Related issues (if any) to this project.
+ - https://github.com/Jash-Shah/pyGnuastro
+mentors:
+# First person in contact; mentors may change before project starts.
+# GitHub handles
+ - mohammad-akhlaghi
```

Comment:
as always, it would be great to have someone would commit being a second mentor.